bindings/python/iota_sdk/utils.py

```diff
@@ -170,8 +169,8 @@ def block_id(block: Block, params: ProtocolParameters) -> HexStr:
         """ Return a block ID (Blake2b256 hash of block bytes) from a block.
         """
         return _call_method('blockId', {
-            'block': block,
-            'protocol_parameters': params,
+            'block': block.to_dict(),
```

Comment:
```suggestion
            'block': block,
```
This shouldn't be necessary

Reply:
that's certainly true now